src/controller_rpc.c

```diff
@@ -965,6 +966,10 @@ controller_commit_actions(clixon_handle           h,
         cprintf(notifycb, "<tid>%" PRIu64"</tid>", ct->ct_id);
         cprintf(notifycb, "<source>actions</source>");
         cprintf(notifycb, "<target>actions</target>");
+
+	if (diff)
+	    cprintf(notifycb, "<diff>true</diff>");
```

Comment:
Need to extend the YANG definition of the services-commit message with a diff boolean element. 
Somewhere here: https://github.com/clicon/clixon-controller/blob/f33a47fe1784476681b2a1127bf0172f15ed59bc/yang/clixon-controller%402024-08-01.yang#L608